requirements.txt

```diff
@@ -15,5 +15,5 @@ distributed<2022.10
 pytest>=3.6,<8.0
 pytest-runner
 pytest-cov
-xdsl>=0.11
+xdsl
```

Comment:
Should we put an upper bound for the future?
What are we using now for testing? (we use the version in https://github.com/xdslproject/devito/pull/46/files#diff-d1bf7976aa4521a259403d1a42af3e2b4d0307f99737f2972b4b5c9c5607896eR41)

so praactically this line here is useless?

Reply:
This line is useless, yes
We could add a few lines in setup.py so as to have a specific commit naturally installed?
A few lines are needed because there is no common syntax for this for pip and setup tools 🫤

Reply:
If you agree, I would try to solve this on its own PR though? I feel it's good enough for now, we at least have CI and a recorded version somewhere.
Let's improve this soon? It's important for users/reproducibility, but a bit sidetracking for development I'd say

Reply:
SOunds good